source/extensions/quic_listeners/quiche/platform/spdy_containers_impl.h

```diff
@@ -1,25 +1,22 @@
 #pragma once
 
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
 #include "absl/container/flat_hash_map.h"
 #include "absl/container/flat_hash_set.h"
 #include "absl/container/inlined_vector.h"
 #include "absl/hash/hash.h"
 
-// The following includes should be:
-//
-// #include "extensions/quic_listeners/quiche/platform/spdy_string_impl.h"
-// #include "extensions/quic_listeners/quiche/platform/spdy_string_piece_impl.h"
 //
 // However, for some reason, bazel.clang_tidy cannot resolve the files when specified this way.
```

Comment:
Should this part of the comment be removed as well?

Reply:
Oh, yes, they should. Done.